core/amber/src/main/protobuf/edu/uci/ics/amber/engine/architecture/worker/controlreturns.proto

```diff
@@ -14,11 +14,17 @@ option (scalapb.options) = {
 message CurrentInputTupleInfo {
 }
 
+message ControlException {
```

Comment:
maybe `ControlHanderException`? Also maybe include an optional field for stack trace?

Reply:
will keep the `ControlException` to be aligned with `ControlCommand` and `ControlReturn`